packages/example-page/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "example-page",
+  "version": "1.0.0",
+  "description": "Example page highlighting the different flavours of substyle",
+  "main": "src/index.js",
+  "author": "Philipp Giese",
+  "license": "MIT",
+  "private": false,
```

Comment:
Should be `true`, no? Do we want to publish this package to npm?

Reply:
Absolutely